apicast/src/provider.lua

```diff
@@ -53,7 +54,8 @@ end
 
 function _M.configured(host)
   local hosts = _M.configuration:find(host)
-  return #hosts > 0
+
+  return next(hosts)
```

Comment:
Converting it to boolean is just another operation that is not needed. 

Here it would return the first index, which would be some service id.

There are some huge discussions in rails over `to_b` in some methods that end with `?` and the decision is to not convert. Lua is like ruby and has the same truthyness/falsyness. 

I'd really prefer this over the `true` variant:

```lua
return next(hosts) and true
```